From 4578726d3b0e5b7c699659c1ecd26194060e7be7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Christoph=20B=C3=B6hmwalder?=
 <christoph.boehmwalder@linbit.com>
Date: Thu, 29 Aug 2024 16:15:47 +0200
Subject: [PATCH 21/32] compat: fixup write_zeroes__no_capable

We now access the field directly instead of using this helper
---
 drbd/drbd-kernel-compat/cocci/write_zeroes__no_capable.cocci | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drbd/drbd-kernel-compat/cocci/write_zeroes__no_capable.cocci b/drbd/drbd-kernel-compat/cocci/write_zeroes__no_capable.cocci
index 4fa6d9e249d7..2b060e2abd16 100644
--- a/drbd/drbd-kernel-compat/cocci/write_zeroes__no_capable.cocci
+++ b/drbd/drbd-kernel-compat/cocci/write_zeroes__no_capable.cocci
@@ -30,5 +30,6 @@ expression e;
 +WARN_ON_ONCE(e); /* WRITE_ZEROES not supported on this kernel */
 
 @@
+struct queue_limits lim;
 @@
-- blk_queue_max_write_zeroes_sectors(...);
+- lim.max_write_zeroes_sectors = ...;
-- 
2.35.3

